superset/models/helpers.py

```diff
@@ -151,6 +151,7 @@ def import_from_dict(
         cls,
         session: Session,
         dict_rep: Dict[Any, Any],
+        dict_keys: Dict[Any, Any] = {},
```

Comment:
Mutable defaults should never be used in Python: https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments